main.js

```diff
@@ -239,5 +240,15 @@ function getScreenshotProtectionLabel () {
     return 'Enable screenshot protection (recommended)'
   }
 }
+
+function registerShortcuts () {
+  electron.globalShortcut.register('CommandOrControl+H', hideAppOnMac)
```

Comment:
Should the process.platform check (ln 249) be moved up here? Otherwise, users can end up with an unnecessary listener.

If that's the case, then it's probably good to rename the hideAppOnMac to just hideApp.

Reply:
@spresnal Yes it should. Gonna change that ASAP.